lib/services/elastic.js

```diff
@@ -479,17 +479,19 @@ function getDocument(index, type, id) {
  * @param  {string} id
  * @param  {object} data
  * @param  {Boolean} refresh
+ * @param  {Boolean} upsert
  * @return {Promise}
  */
-function update(index, type, id, data, refresh = false) { // eslint-disable-line max-params
+function update(index, type, id, data, refresh = false, upsert = false) { // eslint-disable-line max-params
```

Comment:
nit: I like the convention of having boolean variables start with `is`, makes it easier to follow